doc/classes/TextEdit.xml

```diff
@@ -146,6 +150,7 @@
 			<argument index="0" name="keyword" type="String">
 			</argument>
 			<description>
+				Returns the [Color] of a keyword at the specified index.
```

Comment:
This parameter is not an index, but a keyword. Should be:
Returns the [Color] of the specified [code]keyword[/code].

Reply:
I'm confuzed, what is the index argument for then?

Reply:
You are right that its a sting keyword by looking at the code
`Color TextEdit::get_keyword_color(String p_keyword) const {`
but why is there an index argument?

Reply:
There is no index argument provided to the function. The `index` attribute in the XML node `<argument>` is simply the index of the argument itself, as an information for the documentation parser. 
If `<argument index="0">` then this argument is the first parameter of the function.
If `<argument index="1">` then this argument is the second parameter of the function, etc.

Reply:
Sorry my fault 😳